rasa/core/processor.py

```diff
@@ -213,10 +215,39 @@ def get_tracker(self, conversation_id: Text) -> Optional[DialogueStateTracker]:
             conversation.
         """
         conversation_id = conversation_id or DEFAULT_SENDER_ID
+
         return self.tracker_store.get_or_create_tracker(
             conversation_id, append_action_listen=False
         )
 
+    def get_trackers_for_all_conversation_sessions(
+        self, conversation_id: Text,
+    ) -> List[DialogueStateTracker]:
+        """Get all trackers for a conversation.
+
+        Individual trackers are returned for each conversation session found
+        for `conversation_id`.
+
+        Args:
+            conversation_id: The ID of the conversation for which the trackers should
+                be retrieved.
+
+        Returns:
+            Trackers for the conversation.
+        """
+        import rasa.shared.core.trackers
```

Comment:
it's already imported as top level import. A local import doesn't add any benefit then.